docs/geospatial-install.md

```diff
@@ -0,0 +1,95 @@
+
+## Requirements
+To be able to build your custom Docker image it is required to run [Docker](https://www.docker.com/), more precisely the [Docker Engine](https://docs.docker.com/engine/install/).
+Please make yourself familiar with the install instructions according to your operating system.
+In general, you can build the image locally. It is not required that the image is built somewhere on the KNIME Business Hub.
+
+## Dockerfile
+The basis of a Docker image is a Dockerfile. It defines
+- the source image (in this case the official Docker image from KNIME; check out all available images [here](https://docs.knime.com/latest/business_hub_admin_guide/index.html#docker-executor-images)),
+- the libraries/packages to be installed, and subsequently 
+- which extensions should be installed into the executor.
+
+Make sure to save that file with the name ```Dockerfile``` and add no file extension.
+
+Within the Dockerfile, we define two variables at the bottom that are important while installing the extension(s):
+
+```KNIME_UPDATE_SITES``` the URL to the necessary Update Site(s)
+
+```KNIME_FEATURES``` - the Feature ID of the extension to be installed
+
+Check out [this video](https://www.youtube.com/watch?v=-dO79Id3VAo&t=143s) from the KNIMETV channel that helps you to find the right update site URL and the feature ID.
+
+    # Define the base image
+    FROM registry.hub.knime.com/knime/knime-full:r-5.2.1-369
+
+    # Change to root user to be able to install system packages
+    USER root
+
+    # Update/upgrade package manager and install ca-certificates to enable ca certificates that micromamba (for python) is asking for
+    RUN apt-get update && \
+        apt-get upgrade -yq && \
+        apt-get install -yq \
+            ca-certificates && \
+        # cleanup
+        rm -rf /var/lib/apt/lists/*
+
+
+    # Change to knime user to handle extensions
+    USER knime
+
+    # Define the list of update sites and features
+    ENV KNIME_UPDATE_SITES="https://update.knime.com/analytics-platform/5.2,https://update.knime.com/community-contributions/trusted/5.2"
+
+    # Install a feature from the Community Trusted update site
+    ENV KNIME_FEATURES=sdl.harvard.features.geospatial.feature.group
+
+    # Execute extension installation script
+    RUN ./install-extensions.sh
+
+## Build and push the Docker image
+Now, the Docker image can be build from the Dockerfile and made available to the KNIME Business Hub with the following commands.
+
+    # Build the Docker image
+    docker build . -f .\Dockerfile -t knime-full-geo:5.2.1 --no-cache
+
+    # Retag the image to make it readable for KNIME Business Hub
+    docker tag knime-full-geo:5.2.1 registry.<hub-url>/knime-full-geo:5.2.1
```

Comment:
You are right, we call it `<base-url>` in our documentation. We can surely adapt to it, which just makes sense. `<hub-url> probably surfaced here, since it seemed to be less confusing to me.
If necessary, we can add a line here and there, where to execute the respective command.